### PR TITLE
Added speed display

### PR DESCRIPTION
**Pull request by dani909**
I have a project which downloads many files. So I had thought it would be handy to see the current download speed.

The Result:
`Downloading file  41% [==============================>                                          ]  9/23MB (0:00:04 / 0:00:05) (2.4MB/s)`

In case you want my changes you can merge them and if you have any suggestions I would be happy to hear them.

### CODE REVIEW

File: src/main/java/me/tongfei/progressbar/ProgressBar.java [hunk 1]
```diff
@@ -26,15 +26,25 @@
...
-        this(task, initialMax, 1000, System.err, style, "", 1);
+        this(task, initialMax, 1000, System.err, style, "", 1, false);
     }
 
     public ProgressBar(String task, long initialMax, int updateIntervalMillis) {
-        this(task, initialMax, updateIntervalMillis, System.err, ProgressBarStyle.COLORFUL_UNICODE_BLOCK, "", 1);
+        this(task, initialMax, updateIntervalMillis, System.err, ProgressBarStyle.COLORFUL_UNICODE_BLOCK, "", 1, false);
+    }
+
+    public ProgressBar(String task,
```
Comment (ctongfei) [inline src/main/java/me/tongfei/progressbar/ProgressBar.java]: Please do not add new constructors -- should be frozen and customized using ProgressBarBuilders.

File: src/main/java/me/tongfei/progressbar/ProgressBar.java [hunk 2]
```diff
@@ -53,10 +64,11 @@ public ProgressBar(
             PrintStream os,
             ProgressBarStyle style,
             String unitName,
-            long unitSize
+            long unitSize,
+            boolean showSpeed
```
Comment (ctongfei) [inline src/main/java/me/tongfei/progressbar/ProgressBar.java]: Should not modify constructors -- API compatibility is desired.

**Comment by ctongfei** [comment]
Thanks for the contribution! I've merged it in a different branch. After some minor modifications it'll go into the master branch and be released.

**Comment by dani909** [comment]
Ok Thanks!

**Comment by ctongfei** [comment]
Published to `0.7.2`.

**Pull request merged**